tools/jdk/local_java_repository.bzl

```diff
@@ -0,0 +1,69 @@
+"""Java repository implementation
+
+Creates a local repository using jdk.BUILD file.
+
+When java executable is not present it creates a BUILD file with target "jdk"
+displaying an error message.
+"""
+
+def _local_java_repository_impl(repository_ctx):
+    java_home = repository_ctx.attr.java_home
+    java_home_path = repository_ctx.path(java_home)
+    if not java_home_path.exists:
+        fail('The repository\'s path is "%s" (absolute: "%s") ' +
```

Comment:
nit: "the repository" isn't very obvious -- how about "The path indicated by the attribute `java_home`, *PATH*, does not exist"?

Reply:
Done.